### ISSUE TITLE
Managed Network broken after upgrade from 1.1.4 to 1.2.0-pre4-rc10 in Kubernetes Env

### ISSUE THREAD

**Issue by soumyalj** [rancher/rancher#6697]
**Rancher Version:**1.1.4 to 1.2.0-pre4-rc10

**Docker Version:**1.10.3

**OS and where are the hosts located? (cloud, bare metal, etc):**Ubuntu, Cloud

**Setup Details: (single node rancher vs. HA rancher, internal DB vs. external DB)**Single node

**Environment Type: (Cattle/Kubernetes/Swarm/Mesos)**Kuberentes

**Steps to Reproduce:**
1. Create a Kuberentes env in 1.1.4. Create services/pods
2. Upgrade rancher to 1.2.0-pre4-rc10
3. Upgrade the env

**Results:** 
After upgrade, Networking between containers on different hosts is broken. 
etcd containers are unhealthy. etcd logs below: 
```
11/20/2016 2:32:36 PMtime="2016-11-20T22:32:36Z" level=error msg="Error reading metadata version: Get http://rancher-metadata/2015-12-19/version: dial tcp: i/o timeout"
11/20/2016 2:32:37 PM2016-11-20 22:32:37.104077 W | etcdserver: failed to reach the peerURL(http://10.42.42.60:2380) of member 15659f9755403edc (Get http://10.42.42.60:2380/version: dial tcp 10.42.42.60:2380: getsockopt: no route to host)
11/20/2016 2:32:37 PM2016-11-20 22:32:37.104258 W | etcdserver: cannot get the version of member 15659f9755403edc (Get http://10.42.42.60:2380/version: dial tcp 10.42.42.60:2380: getsockopt: no route to host)
11/20/2016 2:32:38 PM2016-11-20 22:32:38.304787 W | etcdserver: failed to reach the peerURL(http://10.42.230.57:2380) of member f3f8a6504f790a90 (Get http://10.42.230.57:2380/version: dial tcp 10.42.230.57:2380: i/o timeout)
11/20/2016 2:32:38 PM2016-11-20 22:32:38.304915 W | etcdserver: cannot get the version of member f3f8a6504f790a90 (Get http://10.42.230.57:2380/version: dial tcp 10.42.230.57:2380: i/o timeout)
11/20/2016 2:32:43 PM2016-11-20 22:32:43.103888 W | etcdserver: failed to reach the peerURL(http://10.42.42.60:2380) of member 15659f9755403edc (Get http://10.42.42.60:2380/version: dial tcp 10.42.42.60:2380: getsockopt: no route to host)
11/20/2016 2:32:43 PM2016-11-20 22:32:43.104020 W | etcdserver: cannot get the version of member 15659f9755403edc (Get http://10.42.42.60:2380/version: dial tcp 10.42.42.60:2380: getsockopt: no route to host)
11/20/2016 2:32:44 PM2016-11-20 22:32:44.304282 W | etcdserver: failed to reach the peerURL(http://10.42.230.57:2380) of member f3f8a6504f790a90 (Get http://10.42.230.57:2380/version: dial tcp 10.42.230.57:2380: i/o timeout)
11/20/2016 2:32:44 PM2016-11-20 22:32:44.304381 W | etcdserver: cannot get the version of member f3f8a6504f790a90 (Get http://10.42.230.57:2380/version: dial tcp 10.42.230.57:2380: i/o timeout)
```

**Comment by deniseschannon** [comment]
Fixed in network-manager - v0.2.6

**Comment by soumyalj** [comment]
Tested with upgrade from 1.1.4 to 1.2.0-pre4-rc11. etcds are functional. Containers on different hosts can talk to each other.
